build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")"
+
+mkdir -p bin
```

Comment:
`go` will create output directories as needed, so you can drop this line.  Confirm with:

```console
$ rm -rf a
$ go build -o a/b/c ./cmd/openshift-install
$ file a/b/c
a/b/c: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked (uses shared libs), not stripped
```